### PR TITLE
[JOLINT-42] created footer for consent-form page 

### PR DESCRIPTION
**Pull request by idaakermark**

### CODE REVIEW

**Review by DavidDyberg — APPROVED** [review]

**Comment by ell-ska** [comment]
testing on development:
everything looks great but i noticed the social links aren’t links. they probably should be!

File: app/consent-form/_components/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  return (
+    <footer className='space-y-8 bg-blue-dull px-32 pb-8 pt-20 text-neutral-100'>
+      <div className='flex justify-between'>
+        <div className='space-y-4'>
+          <p>Jolint Network Analytics AB</p>
+          <p>Org. Nr.559324-1770</p>
+          <p>Gothenburg, Sweden</p>
+        </div>
+        <div className='space-y-4'>
+          <p className='flex gap-4'>
```
Comment (ell-ska) [inline app/consent-form/_components/Footer.tsx:13]: the social links should be <Link>

**Review by ell-ska — CHANGES_REQUESTED** [review]

File: app/consent-form/_components/Footer.tsx (hunk 1 above)
Comment (ell-ska) [reply app/consent-form/_components/Footer.tsx:13]: links

**Comment by ell-ska** [comment]
testing on development:
good!

**Review by ell-ska — APPROVED** [review]

**Pull request merged**